src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
```

Comment:
you can make a destructuring from the comment
```suggestion
export const CommentInfo = ({ comment }) => {
  const { .. } = comment;
  
  return (
```